include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -330,427 +330,1283 @@ struct SPhysicalDeviceFeatures
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_AMD_extension_32
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_AMD_extension_33
+
+    // [DEPRECATED] Vulkan core 1.2
+    /* VK_AMD_draw_indirect_count */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_AMD_extension_35
+
+    // [DEPRECATED] Promoted to VK_KHR_maintenance1, non-optional core VK 1.1
+    /* VK_AMD_negative_viewport_height */
+
+    // [EXPOSE AS LIMIT] The only reason we still keep it around is because it provides FP16 trig and special functions
+    /* VK_AMD_gpu_shader_half_float */
+
+    // [DEPRECATED] Superseded by KHR_shader_subgroup_ballot
+    /* VK_AMD_shader_ballot */
+
+    // [DO NOT EXPOSE] Don't want to be on the hook for the MPEG-LA useless Patent Trolls
+    /* VK_EXT_video_encode_h264 */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_video_encode_h265
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_video_decode_h264
+
+    // [TODO LATER] Won't expose for now, API changes necessary
+    /* VK_AMD_texture_gather_bias_lod */
+
+    // [TODO] need impl, also expose as a limit?
+    /* VK_AMD_shader_info */
+    bool shaderInfoAMD = false;
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_AMD_extension_44
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_dynamic_rendering
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_AMD_extension_46
+
+    // Enabled by Default, Moved to Limits
+    /* VK_AMD_shader_image_load_store_lod */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_NVX_extension_48
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_GOOGLE_extension_49
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_GGP_stream_descriptor_surface
+
+    // [DO NOT EXPOSE] for a very long time
+    /* CornerSampledImageFeaturesNV *//* VK_NV_corner_sampled_image */
+    //bool           cornerSampledImage;
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_NV_private_vendor_info
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_NV_extension_53
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_multiview
+
+    // [DO NOT EXPOSE] Enables certain formats in Vulkan, we just enable them if available or else we need to make format support query functions in LogicalDevice as well
+    /* VK_IMG_format_pvrtc */
+
+    // [DEPRECATED] by VK_KHR_external_memory_capabilities
+    // VK_NV_external_memory_capabilities
+
+    // [DEPRECATED] by VK_KHR_external_memory
+    // VK_NV_external_memory
+
+    // [DEPRECATED] Promoted to VK_KHR_external_memory_win32 
+    /* VK_NV_external_memory_win32 */
+
+    // [DEPRECATED] Promoted to VK_KHR_win32_keyed_mutex 
+    /* VK_NV_win32_keyed_mutex */
+
+    // [DEPRECATED] Promoted to non-optional core VK 1.1
+    /* VK_KHR_get_physical_device_properties2 */
+
+    // [DEPRECATED] Promoted to non-optional core VK 1.1
+    /* VK_KHR_device_group */
+    /* VK_KHR_device_group_creation */
```

Comment:
`VK_KHR_device_group_creation ` in duplicate then the comment at the later `VK_KHR_device_group_creation ` is wrong